packages/core/src/services/documents/create.ts

```diff
@@ -22,6 +28,7 @@ export async function createNewDocument({
 
     const workspace = await findWorkspaceFromCommit(commit, tx)
     const docsScope = new DocumentVersionsRepository(workspace!.id, tx)
+    const path = sanitizeDocumentPath(inputPath)!
```

Comment:
example of why validations at service level is a bad idea. Besides services having multiple responsibilities, I can't confidently refactor and add a validation to the model on every write. Is updating the create/update services enough? are the models written somewhere else?